docs/api_reference/src/quantities.tex

```diff
@@ -9,15 +9,26 @@
 
 \begin{modularlibsumtab}{Quantities library summary}{qties.summary}
 \ref{qty.helpers}       & Helpers                                & \tcode{mp_units.core} \\
-\ref{qty.traits}        & Traits                                 & \\
-\ref{qty.concepts}      & Concepts                               & \\
-\ref{qty.types}         & Types                                  & \\
-\ref{qty.compat}        & Compatibility                          & \\
-\ref{qty.one}           & Dimension one                          & \\ \rowsep
+\ref{qty.expr.temp}     & Expression template                    & \\
+\ref{qty.dim}           & Dimension                              & \\
+\ref{qty.spec}          & Quantity specification                 & \\
+\ref{qty.unit.mag}      & Unit magnitude                         & \\
+\ref{qty.unit}          & Unit                                   & \\
+\ref{qty.ref}           & Reference                              & \\
+\ref{qty.rep}           & Representation                         & \\
+\ref{qty.traits}        & Quantity traits                        & \\
+\ref{qty}               & Class template \tcode{quantity}        & \\
+\ref{qty.pt.orig}       & Point origin                           & \\
+\ref{qty.pt}            & Class template \tcode{quantity_point}  & \\ \rowsep
 \ref{qty.systems}       & Systems                                & \tcode{mp_units.systems} \\
 \ref{qty.chrono}        & \tcode{std::chrono} compatibility      & \\
```

Comment:
Sure, "Unit magnitude" should be under "Unit".

Expression template probably as well. We may also need to rename it, as some people already objected to the use of this term for our purposes. The general-purpose expression template library would have totally different interfaces and rules. Ours is DSL (Domain Specific) version of expression templates.

> Also, shouldn't these concepts be exposition only?

Initially, I thought it might be useful for a user to assert if a custom type satisfies those requirements. This is also why `RepresentationOf` still takes `quantity_character` explicitly (even though not used by the framework anywhere). If you think it is not a good enough reason to make it public, we can convert it to an exposition-only concept.

Reply:
> Expression template probably as well. We may also need to rename it, as some people already objected to the use of this term for our purposes. The general-purpose expression template library would have totally different interfaces and rules. Ours is DSL (Domain Specific) version of expression templates.

To me, it seems more like a [computer algebra system](https://en.wikipedia.org/wiki/Computer_algebra_system).

Reply:
> It should be possible to dissolve 5.13 "Quantity values".
> `quantity_values` can be moved under 5.12 "Representation".
> And if it isn't added under `::std`, it can then be renamed to `representation_values`
> (e.g., `::std::units::representation_values`).
> `QuantityLike` and `QuantityPointLike` can then be moved
> next to `quantity` and `quantity_point`, respectively.

You can already do this rename, though.

Reply:
Done

Reply:
> To me, it seems more like a [computer algebra system](https://en.wikipedia.org/wiki/Computer_algebra_system).

I am not sure. It does not feel like a calculator 😉

Reply:
How about 5.6 "Symbolic computations"?

Reply:
We do not compute the result. We just process and simplify. Maybe "Symbolic expressions" is a better name?

Also, with this, we will not have to refactor all the functions starting with `expr_XXX` already :wink: Only the header file would change.

Reply:
![1732711172](https://github.com/user-attachments/assets/1ad4085c-9ef4-42e9-a797-bff2d370a4d7)

<details><summary>PDF tree view</summary>

![1732711328](https://github.com/user-attachments/assets/45af8210-51e7-46b6-84a3-861aff2974d1)

</details>

Now it looks like this (not yet pushed).
It'd be possible to go further and group 5.6--5.9 all under 5.6 "Reference".
The current 5.9 "Reference" can be renamed "class template `reference`", since it's such a simple and short subclause.
It would look like this:

![1732711552](https://github.com/user-attachments/assets/0bd053da-dbe9-4ec6-aafd-351f8858065b)

<details><summary>PDF tree view</summary>

![1732711571](https://github.com/user-attachments/assets/00827c5c-d87e-4785-b39d-66636c332ac3)

</details>

Reply:
I like the second tree more 😃 

There are a few things that still could be improved:
- I am not sure if "Helpers" is the right name for this chapter
- "Point origin" and "Class template `quantity_point`" could be merged as well
- `std::chrono` compatibility should probably be renamed to [something more general](https://mpusz.github.io/mp-units/latest/users_guide/use_cases/interoperability_with_other_libraries/)

Reply:
> `std::chrono` compatibility should probably be renamed to [something more general](https://mpusz.github.io/mp-units/latest/users_guide/use_cases/interoperability_with_other_libraries/)

I don't think so.
The chapter is specifically about specializing the -like-traits templates for `std::chrono`.

Reply:
Those traits can be used with any third-party library. There are also `XXXLike` concepts associated with them.

Reply:
> I am not sure if "Helpers" is the right name for this chapter

How about "Utilities"?

Reply:
![1732723351](https://github.com/user-attachments/assets/30ce917e-fb28-487f-85df-29e55efb04ce)

<details><summary>PDF tree view</summary>

![1732723384](https://github.com/user-attachments/assets/1523fb86-c820-40e5-b3a7-893d068b998d)

</details>

Now it looks like this (not yet pushed).
Note that we have 3 module synopses instead of 1 header synopsis.
For the latter, this would actually go from 5.3--5.9.

Reply:
<details><summary>Current PDF tree view</summary>

![1732850387](https://github.com/user-attachments/assets/08701431-622c-4185-a279-30fe4addea04)
![1732850403](https://github.com/user-attachments/assets/ffbe63a7-72a5-430f-9481-649e0fd5e567)

</details>

Reply:
"Of dimension one" seems awkward to me. Is it only about `dimensionless`? If so I would prefer to use `dimensionless` in the name. Alternatively, a full name, "Quantity of dimension one", should probably be provided.

Notice that we have many specialized quantities of a kind `dimensionless` (e.g., `angular_measure`). All of them have dimension one.